src/index.ts

```diff
@@ -3,6 +3,7 @@ import {
   MessageChannel,
   MessagePort,
   receiveMessageOnPort,
+  TransferListItem,
```

Comment:
Turns out `worker_threads` exports a `TransferListItem` type for us, so we don't have to extract it from `MessagePort.postMessage` anymore :)